docs/source/conf.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import inspect
+import os
+import raiwidgets
+import sys
+sys.path.insert(0, os.path.abspath(os.path.join('..', 'raiwidgets')))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Responsible AI Widgets'
+copyright = 'Microsoft Corporation'
+author = 'Roman Lutz, Ilya Matiach, Ke Xu'
+
+# The full version, including alpha/beta/rc tags
+release = 'v0.2.0'
```

Comment:
this is another place where we need to keep track of version and could reuse the same constant.  actually this should be 0.2.1 now.

Reply:
Ah, here I should be able to just reference the other one.